packages/node_modules/@ciscospark/react-component-ringtone/src/index.js

```diff
@@ -3,7 +3,7 @@ import {Component, PropTypes} from 'react';
 import {Howl} from 'howler';
 
 import RingtoneIncoming from './media/Ringtone.mp3';
-import RingtoneRingback from './media/Ringback_v7_norm.wav';
+import RingtoneRingback from './media/Ringback_v2.mp3';
```

Comment:
got it from the web client. It's 1/5 the size of the wav so I figure it's better than delivering a 750kb file to the user.